src/katello/deb_tracer.py

```diff
@@ -0,0 +1,73 @@
+from __future__ import absolute_import
+from os import path
+import subprocess
+import sys
+try:
+    from shutil import which
+except ImportError:  # on Python 2
+    from distutils.spawn import find_executable as which
+
+# The file is created by /etc/kernel/postinst.d/unattended-upgrades (part of unattended-upgrades pkg)
+REBOOT_NEEDED_FLAG = "/var/run/reboot-required"
+
+
+class AptTracerApp:
+    def __init__(self, name, helper, type):
+        self.name = name
+        self.helper = helper
+        self.type = type
+
+
+def has_needrestart():
+    if sys.version_info[0] < 3:
+        return which("needrestart") is not None
+    else:
+        return which("needrestart") is not None
```

Comment:
where is the difference between those two?

Reply:
There is no difference. I pushed a fixup commit.